app/clients/freshdesk.py

```diff
@@ -1,12 +1,84 @@
 import json
-
 import requests
+
 from requests.auth import HTTPBasicAuth
+from typing import Dict, List, Union
 
 from flask import current_app
 
+from app.user.contact_request import ContactRequest
+
+
+__all__ = [
+    'Freshdesk'
+]
+
 
 class Freshdesk(object):
+
+    def _generate_description(self):
+        message = ''
+        if 'demo' in self.contact.support_type.lower():
```

Comment:
It may be good to add a helper method in `ContactRequest` for this

Reply:
sorry I don't understand what would the helper method do?
I do not want to tie contactrequest to the context of zendesk or freshdesk - I'd like to understand this more